R/mod_body_coc_competition.R

```diff
@@ -0,0 +1,475 @@
+#' body_coc_competition UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+mod_body_coc_competition_ui <- function(id){
+  ns <- NS(id)
+  tagList(
+    ui_picker_program(
+          inputId = ns("pe_provider"),
+          label = "Select your CoC-funded Provider",
+          choices = sort(pe_summary_validation()$AltProjectName) |> unique(),
+          selected = pe_summary_validation()$AltProjectName[1],
+          options = shinyWidgets::pickerOptions(liveSearch = TRUE,
```

Comment:
I do believe these pickerOptions are enabled by default in `ui_picker_program`. Be sure to familiarize with the default arguments (`?ui_picker_program`) for a function so as not to write code that doesn't need to be written.